pandas/tests/series/indexing/test_datetime.py

```diff
@@ -667,3 +667,14 @@ def test_setitem_tuple_with_datetimetz():
     result[(0, 1)] = np.nan
     expected.iloc[0] = np.nan
     tm.assert_series_equal(result, expected)
+
+
+def test_indexing_regression():
+    # Issue 34860
+    arr = date_range("1/1/2008", "1/1/2009")
+    ser = arr.to_series()["2008"]
```

Comment:
can you call this result

Reply:
Sure, I can move the test to pandas/tests/indexing/test_partial.py location.
Could not understand your another comment. Could you please explain.

Reply:
Got it, you mean rename `ser` to `result`. I will do that as well.